src/main/java/com/linkedin/kmf/services/CommitLatencyMetrics.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.linkedin.kmf.services;
+
+import java.util.Map;
+import org.apache.kafka.common.MetricName;
+import org.apache.kafka.common.metrics.Metrics;
+import org.apache.kafka.common.metrics.Sensor;
+import org.apache.kafka.common.metrics.stats.Avg;
+import org.apache.kafka.common.metrics.stats.Max;
+import org.apache.kafka.common.metrics.stats.Percentile;
+import org.apache.kafka.common.metrics.stats.Percentiles;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * The CommitLatencyMetrics class contains methods that measures and
+ * determines the latency of Kafka consumer offset commit().
+ */
+public class CommitLatencyMetrics {
+  private static final String METRIC_GROUP_NAME = "commit-latency-service";
+  private static final Logger LOG = LoggerFactory.getLogger(CommitLatencyMetrics.class);
+  final Sensor _commitOffsetLatency;
```

Comment:
Can you make this field private and provide a getter method for it? I think in Java, it is always a good practice to make fields private and provides getters and/or setters for them if necessary. The reasons are:

1. Make the class more testable since the class would be easier for mocking
2. We can control whether we return a reference to the original field (the real guts) or make a copy of the object and return a reference to the copied object.
3. It opens up opportunity for us to instrument the getter or setter code path. For example, how many times some getter of some fields has been called and etc.